packages/playground-ui/src/ds/components/CommandPalette/command-palette.css

```
@@ -0,0 +1,133 @@
+.command-palette-popup[data-open] {
+  animation: none !important;
+}
+
+.command-palette-popup[data-closed] {
+  animation: command-palette-popup-hold 260ms linear both !important;
+}
+
+.command-palette-surface {
+  --command-palette-enter-delay: 0ms;
+  --command-palette-exit-delay: 0ms;
+  transform-origin: center;
```

Stylelint (17.14.1)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)